### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by rintcius**
Walk through Functor documentation.

### CODE REVIEW

File: docs/src/main/tut/functor.md [hunk 1]
```diff
@@ -55,44 +55,41 @@ implicit def function1Functor[In]: Functor[Function1[In, ?]] =
...
-This compiler plugin lets us more easily change the number of type
-holes a type has. In this case, we took a type which normally has two
-type holes, `Function1` and filled one of the holes, leaving the other
-hole open. `Function1[In,?]` has the first type parameter filled,
-while the second is still open. Without kind-projector, we'd have to
-write this as something like: `({type F[A] = Function1[In,A]})#F`,
-which is much harder to read and understand.
-
-## Using functor
+This example demonstrates the use of the
```
Comment (rintcius) [inline docs/src/main/tut/functor.md]: remove some duplication in this paragraph

File: docs/src/main/tut/functor.md [hunk 2]
```diff
@@ -55,44 +55,41 @@ implicit def function1Functor[In]: Functor[Function1[In, ?]] =
...
+This compiler plugin can help us when we need to change the number of type
+holes. In the example above, we took a type which normally has two type holes, 
+`Function1[?,?]` and constrained one of the holes to be the `In` type, 
+leaving just one hole for the return type, resulting in `Function1[In,?]`. 
+Without kind-projector, we'd have to write this as something like 
+`({type F[A] = Function1[In,A]})#F`, which is much harder to read and understand.
+
+## Using Functor
 
 ### map
```
Comment (rintcius) [inline docs/src/main/tut/functor.md]: I moved List example before Some because List is a bit easier to explain/show

**Comment by non** [comment]
These are great edits, thanks! I'm ready to merge this now -- do you have other edits to make?

:+1:

**Comment by rintcius** [comment]
No, no more edits at the moment

**Pull request merged**